pkg/controllers/networkpolicy/networkpolicy_controller.go

```diff
@@ -135,6 +137,14 @@ func (r *NetworkPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 func (r *NetworkPolicyReconciler) setupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&networkingv1.NetworkPolicy{}).
+		Watches(
```

Comment:
It looks like we also miss the namespace handler to handle the namespace label change event. 
It can be enhanced in another patch.

Reply:
added